cmd/fakemachine/main.go

```diff
@@ -18,6 +19,7 @@ type Options struct {
 	CPUs        int               `short:"c" long:"cpus" description:"Number of CPUs for the fakemachine"`
 	ScratchSize string            `short:"s" long:"scratchsize" description:"On-disk scratch space size (with a unit suffix, e.g. 4G); if unset, memory backed scratch space is used"`
 	ShowBoot    bool              `long:"show-boot" description:"Show boot/console messages from the fakemachine"`
+	KernelPath  string            `short:"k" long:"kernel" description:"kernel to use"`
```

Comment:
When you say "kernel to use", is that just the filename (within the usual search paths), the complete absolute path + filename, or the path relative to `RECIPE_DIR` + filename?

Please enhance the help message so it's clear which one we're talking about. Also do update the README.md thanks o/